tests/javaRosa.js

```diff
@@ -518,16 +518,6 @@ require([
             assert.equal(hinLabel[0].selectionEnd, 15);
         });
 
-        _.each({group: GROUP_HELP_XML, select: SELECT1_HELP_XML}, function (XML, name) {
```

Comment:
You removed two tests here. I think we want to keep the second one (for `select`). Also, static/javaRosa/group-help.xml should probably be removed as well.